### PR TITLE
Add missing entries in the OPTIONS section

### PR DESCRIPTION
**Pull request by uniontech-lilinjie**
Add missing options profile_info, auto_profile and profile mode in the OPTIONS section of the README.
Signed-off-by: lilinjie <lilinjie@uniontech.com>

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).
 To disable all tunings, run:
 # tuned-adm off
 
+To show information/description of given profile or current profile if no profile is specified, run:
+# tuned-adm profile_info
+
+To verifies current profile against system settings, run:
```
Comment (yarda) [inline README:52]: I think this should spell either:
'To verify....' or 'Verifies...', but if there is ', run:', I think it should be 'To verify:'

File: README [hunk 2]
```diff
@@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).
...
+# tuned-adm verify
+
+Enable automatic profile selection mode, switch to the recommended profile, run:
```
Comment (yarda) [inline README:55]: If there is ', run:' I think it should be 'To enable...', or drop the ', run'.

File: README [hunk 3]
```diff
@@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).
...
+# tuned-adm auto_profile
+
+Show current profile selection mode, run:
```
Comment (yarda) [inline README:58]: If there is ', run:' I think it should be 'To show...', or drop the ', run'.

**Review by yarda — CHANGES_REQUESTED** [review]
Thanks for the PR, I think some changes are needed for consistency.

**Comment by uniontech-lilinjie** [comment]
&nbsp;hello, I revised it as you suggested (PR #456)

&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Wed, Sep 7, 2022 06:20 AM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [redhat-performance/tuned] Add missing entries in the OPTIONS section (PR #456)

&nbsp;



 
@yarda requested changes on this pull request.
 
Thanks for the PR, I think some changes are needed for consistency.

 
In README:
 &gt; @@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).  To disable all tunings, run:  # tuned-adm off   +To show information/description of given profile or current profile if no profile is specified, run: +# tuned-adm profile_info + +To verifies current profile against system settings, run:  
I think this should spell either:
 'To verify....' or 'Verifies...', but if there is ', run:', I think it should be 'To verify:'
 
 
In README:
 &gt; @@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).  To disable all tunings, run:  # tuned-adm off   +To show information/description of given profile or current profile if no profile is specified, run: +# tuned-adm profile_info + +To verifies current profile against system settings, run: +# tuned-adm verify + +Enable automatic profile selection mode, switch to the recommended profile, run:  
If there is ', run:' I think it should be 'To enable...', or drop the ', run'.
 
 
In README:
 &gt; @@ -46,6 +46,18 @@ choice is used when the daemon is restarted (e.g. with the machine reboot).  To disable all tunings, run:  # tuned-adm off   +To show information/description of given profile or current profile if no profile is specified, run: +# tuned-adm profile_info + +To verifies current profile against system settings, run: +# tuned-adm verify + +Enable automatic profile selection mode, switch to the recommended profile, run: +# tuned-adm auto_profile + +Show current profile selection mode, run:  
If there is ', run:' I think it should be 'To show...', or drop the ', run'.

**Pull request merged**